### PR TITLE
Added Logging and exception handling

### PR DESCRIPTION
**Pull request by omkar-ti**

### CODE REVIEW

File: fast_api_als/database/db_helper.py [hunk 1]
```diff
@@ -343,15 +368,20 @@ def update_lead_conversion(self, lead_uuid: str, oem: str, converted: int):
...
-def verify_response(response_code):
-    if not response_code == 200:
-        pass
-    else:
-        pass
 
+def log_db_table_operations(res, func1, func2):
+    response_code = res['ResponseMetadata']['HTTPStatusCode']
+    if response_code == 200:
+        logging.info("[DB Helper] HTTP code returned = %d for operation %s called in function %s",response_code, func1.__name__, func2.__name__)
```
Comment (nitish-ti) [inline fast_api_als/database/db_helper.py:379]: If you're only using the function's name, then why are you using the name of the function as an argument.
It misleads anyone who is reading your code, it's completely unnecessary to pass func here.

File: fast_api_als/routers/submit_lead.py [hunk 2]
```diff
@@ -55,6 +56,7 @@ async def submit(file: Request, apikey: APIKey = Depends(get_api_key)):
         }
         item, path = create_quicksight_data(obj, 'unknown_hash', 'REJECTED', '1_INVALID_XML', {})
         s3_helper_client.put_file(item, path)
+        logging.error("[Submit Lead] XML Rejected. Code - 1_INVALID_XML.\nThe XML -\n%s",body)
```
Comment (nitish-ti) [inline fast_api_als/routers/submit_lead.py]: using `[Submit Lead]` in every log statement is a bad practice.

File: fast_api_als/database/db_helper.py (hunk 1 above)
Comment (nitish-ti) [inline fast_api_als/database/db_helper.py]: If you're only using the function's name, then why are you using the name of the function as an argument.
It misleads anyone who is reading your code, it's completely unnecessary to pass func here.

Comment (omkar-ti) [reply fast_api_als/database/db_helper.py]: Understood. I have fixed it.

File: fast_api_als/routers/submit_lead.py (hunk 2 above)
Comment (omkar-ti) [reply fast_api_als/routers/submit_lead.py]: I have fixed it. Changed the formatting of the logs.

**Pull request closed without merging**